### DISCUSSION TITLE
Enabling/disabling leaderboards mid play pop-up

### DISCUSSION THREAD

**Discussion by Catzzye** [ppy/osu#26869] [Ideas]
Hello,
It would be a good quality-of-life feature if the pop-up for turning on/off the leaderboard during actual play would not display, since it can be distracting.

![osu_2024-01-31_07-29-21](https://github.com/ppy/osu/assets/16815295/7843999a-94e6-4b3e-873b-55d20cd71ab6)

What do you guys think?
_I don't see any similar topics under Ideas I hope it's not a duplicate :)_

**Comment by Stoppedpuma** [comment]
The toast will be getting a redesign #25094 when designs are ready and will hopefully be something less in the way / distracting. Making the toast 50-80% transparent during gameplay would be a nice change though so there's still a little feedback.
